### ISSUE TITLE
Abnormal test result (HP:0500014)

### ISSUE THREAD

**Issue by psergouniotis** [obophenotype/human-phenotype-ontology#3363]
Having this term has major implications. For example should all eye test related terms be added as subclasses? 

Also should there be terms like "Abnormal sign" and "Abnormal symptom"?

**Comment by pnrobinson** [comment]
@drseb @mellybelly @cmungall 
I am not sure it was the best idea to add the term of Abnormal test result -- this came up after our cardiology workshop. Panos, your question is good. I do not think that having extremely general terms liuke "Abnormal sign" would be very useful. I am not really sure whether it would not be better to have something like "Abnormal XYZ test result" classes, where XYZ is something like cardiology or ophthalmology. This could be at the same level as abnormal morphology and abnormal physiology.

**Comment by drseb** [comment]
Hmm. I agree. We should something like an "advisory board" that is consulted, when terms at level one or two are to be added. It feels to me like almost everything belongs under test-result. Maybe I am wrong and there is a extremely fantastic definition of the word "test", but at the moment I think it is better to revert this.

**Comment by pnrobinson** [comment]
@drseb do you also agree to having an Abnormal cardiovascular test result (and etc)? I have mixed feelings about this, but the cardiologists seemed to think it would be very useful.
@mellybelly @cmungall I will revert this unless you have objections -- please comment this week!

**Comment by drseb** [comment]
I agree that this seems to make more sense, but I would still be interested in everybody's opinion on the definition of "test result". What HPO classes does it include and what does it exclude. To me it feels like a lot the classes in HPO are in some kind of abnormal result of some kind of test... but maybe I am totally on the wrong track here.

**Comment by mellybelly** [comment]
How about it be 'abnormal laboratory assay' or 'abnormal laboratory test result' ?  "test result" by itself seems too generic to me.

Also, we should be able to infer the subclasses of this term if these are logically defined based upon OBI assays or LOINC. Do we have a design pattern for this yet?

**Comment by psergouniotis** [comment]
I am clearly not an expert in this but I just wanted to clarify my opinion:

 I totally agree that having extremely general terms like "Abnormal sign", "Abnormal symptom", "Abnormal test" and "Abnormal laboratory test" would not be useful (esp for my field) and will create an issue with inferring subclasses. 

If cardiologists want to have "Abnormal cardiovascular test result" and there are reasonable arguments for this, then they can have it. However, making a term "Abnormal test result (HP:0500014)" confuses things and is perhaps unnecessary. 

I think that what is "test" and what is "laboratory test" is not directly intuitive. 

In summary, I would propose suppressing the term "Abnormal test result (HP:0500014)" and add its current subclasses to a relevant place in the cardiology hierarchy (as other specialties are doing).

**Comment by LOURGHI** [comment]
I thought that this superclass was going to be used for abnormal results of tests carried out to evaluate the response of an organ to a physical stimulation like for phototest  or chemical as for the endocrine tests as well as the diagnostic test like positive tuberculin skin test , positive skin test.

**Comment by pnrobinson** [comment]
I agree with Seb and Panos that it is not useful to have a separate high-level class for test result. I have place the cardiac tests under cardiology, and flow cytometry under cellular phenotype. We can continue to place tests "where they belong".
